static/css/character_card_manager.css

```
@@ -1104,31 +1127,44 @@ min-height: 0;
-    padding: 10px 0;
+    font-size: 15px;
+    font-weight: 800;
+    letter-spacing: 0.4px;
+    padding: 8px 14px;
     cursor: pointer;
     text-shadow: 0px 1px 2px rgba(0, 0, 0, 0.1);
     transition: filter 0.2s;
     overflow: hidden;
+    line-height: 1.2;
+    text-align: center;
+    word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)